query-result-set.hh

```diff
@@ -52,6 +52,12 @@ public:
         : _schema{schema}
         , _cells{std::move(cells)}
     { }
+    result_set_row(result_set_row&&) = default;
+    result_set_row(const result_set_row&) = delete;
+    result_set_row& operator=(const result_set_row&) = delete;
+    result_set_row duplicate() const {
```

Comment:
Usual name for these kinds of methods is `clone` or `copy`, best not to introduce new terms for such a commonly used method.